addon/components/bs-modal.js

```diff
@@ -193,6 +193,18 @@ export default class Modal extends Component {
   @arg
   scrollable = false;
 
+  /**
+   *  Allows adding fullscreen mode for modals. It will
+   *  apply the `modal-fullscreen' class when using `true` and
+   *  `modal-fullscreen-[x]-down` class when using BS breakpoints
+   *   ([x] = 'sm', 'md', 'lg', 'xl', 'xxl').
+   *
+   * @property fullscreen
+   * @type {(Boolean|String)}
+   * @default true
```

Comment:
I think the default value is either `undefined` or `false`. If it would be `true`, the modal would be shown if `<BsModal>` is invoked without setting `@fullscreen` argument.

I guess documenting it as `false` provides most clarity for readers. That the default value actually used is `undefined`, seems to be an implementation detail on our side an user does not need to care about.

```suggestion
   * @default false
```

Reply:
Fixed